src/arch/riscv/isa.cc

```diff
@@ -779,6 +780,11 @@ ISA::setMiscReg(RegIndex idx, RegVal val)
                 setMiscRegNoEffect(idx, new_val);
             }
             break;
+          case MISCREG_SENVCFG:
+            {
+                setMiscRegNoEffect(idx, val + 1);
```

Comment:
We should probably make this behave correctly before committing. I'll see if I have time today to look at it.